koap/src/commonMain/kotlin/Encoder.kt

```diff
@@ -1,3 +1,4 @@
+@file:JsExport
```

Comment:
ultra ultra ultra nitpick

```suggestion
@file:JsExport

```